dlt/destinations/impl/redshift/redshift.py

```diff
@@ -76,15 +78,23 @@ def run(self) -> None:
         credentials = ""
         if self._staging_iam_role:
             credentials = f"IAM_ROLE '{self._staging_iam_role}'"
+            if self._region_name:
+                credentials += f" REGION '{self._region_name}'"
         elif self._staging_credentials and isinstance(
             self._staging_credentials, AwsCredentialsWithoutDefaults
         ):
             aws_access_key = self._staging_credentials.aws_access_key_id
             aws_secret_key = self._staging_credentials.aws_secret_access_key
+            region_name = self._staging_credentials.region_name
             credentials = (
                 "CREDENTIALS"
-                f" 'aws_access_key_id={aws_access_key};aws_secret_access_key={aws_secret_key}'"
+                f" 'aws_access_key_id={aws_access_key};aws_secret_access_key={aws_secret_key}"
             )
+            if region_name:
+                credentials = (
+                    "CREDENTIALS"
+                    f" 'aws_access_key_id={aws_access_key};aws_secret_access_key={aws_secret_key};region={region_name};'"
```

Comment:
if region is unspecified, than there shouldnbt a "region=" at the end of the string (or tempvar)

Reply:
but that's what I am doing if region_name is unspecified it will use the query template mentioned above. that's why I initially thought it would be easier to append the region if it is specified and to use two "different" templates :)

Reply:
ok, lets not do the separate templateVar and just attach the region to the credentials